File: DSA/SakshamDwivedi_590013147/Day32/D32P3.py
#Solution Function
def xor_all(arr):
    final_xor = 0
    for i in arr:
        final_xor ^= i
    return final_xor

#Fixed Input
print(xor_all([1, 2, 3]))

#Custom Input
arr = list(int(x) for x in input("Enter the array : ").split())
print(xor_all(arr))

File: DSA/SakshamDwivedi_590013147/Day32/D32P3.c
#include <stdio.h>

int size;
int arr[100];
int final_xor = 0;

int main(){
  printf("Enter number of elements : ");
  scanf("%d", &size);
  printf("Enter the array : ");
  for(int i = 0; i < size; i++){
    scanf("%d", &arr[i]);
  }
  for(int i = 0; i < size; i++){
    final_xor ^= arr[i];
  }
  printf("Total XOR : %d", final_xor);

  return 0;
}
